app/views/articles/_user_metadata.html.erb

```diff
@@ -1,6 +1,6 @@
 <style>
   .user-metadata-details-inner a {
-    color: <%= HexComparer.new([user_colors(@user)[:bg], user_colors(@user)[:text]]).brightness(0.95) %>
+    color: var(--body-color);
   }
 </style>
```

Comment:
Readability issue here happens (and is being reported) often enough to consider it harm to customize text colors with profile/brand colors. So my thinking is we should NOT use profile/brand colors for any texts. Instead profile/brand colors should only be used as decorations so they never ever affect readability of important information.

That being said I suggest we remove this entire custom styling here instead of replacing link color with `var(--body-color)` - it's not ideal because we actually DO want to differentiate links within text.